cypress/integration/plugins/customImportMap/0_before.spec.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+
+import { testIndexDataSet, BASE_PATH } from '../../../utils/constants';
+
+describe('Before', () => {
+  before(() => {
+    cy.deleteAllIndices();
+
+    localStorage.setItem('home:welcome:show', 'true');
+
+    cy.visit(`${BASE_PATH}/app/home#/tutorial_directory/sampleData`);
+
+    // Click on "Sample data" tab
+    cy.contains('Sample data').click({ force: true });
+    // Load sample flights data
+    cy.get(`button[data-test-subj="addSampleDataSetflights"]`).click({
+      force: true,
+    });
+
+    // Verify that sample data is add by checking toast notification
+    cy.contains('Sample flight data installed', { timeout: 60000 });
+
+    cy.visit(`${BASE_PATH}/app/visualize#/`);
+
+    // Click on "Create Visualization" tab
+    cy.contains('Create visualization').click({ force: true });
+
+    // Click on "Region Map" icon
+    cy.contains('Region Map').click({ force: true });
+
+    // Select index source - [Flights] Flight Log
+    cy.contains('[Flights] Flight Log').click({ force: true });
+
+    // Click on "Import Vector Map" tab, which is part of customImportMap plugin
+    cy.contains('Import Vector Map').click({ force: true });
```

Comment:
should something happen after clicking `Import Vector Map`?

Reply:
So, Import Vector Map shows up only if the customImportMap plugin is installed. It's the first check we can have in place to verify the plugin installation.